doc/python/axes.md

```diff
@@ -464,6 +488,21 @@ fig.update_yaxes(showgrid=True, gridwidth=1, gridcolor='LightPink')
 fig.show()
 ```
 
+_new in 5.8_
+
+By default grid lines are `solid`. Set the `griddash` property to change this style. In this example we display the x-axis grid lines as `dot`. It can also be set to `dash`, `longdash`, `dashdot`, or `longdashdot`.
```

Comment:
In addition to those predefined line types, one could use custom values for `griddash` e.g. `"10px"` | more info: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray

Reply:
@archmoj, should it work like this? That throws an error
```fig.update_xaxes(showgrid=True, gridwidth=1, gridcolor='LightPink', griddash='10px')```

Reply:
What is the error message?
Here is a [codepen](https://codepen.io/MojtabaSamimi/pen/KKQdKGL?editors=0010) you could may adapt which also illustrate the case of "monthly period labels with weekly minor ticks".

Reply:
Created an issue for that https://github.com/plotly/plotly.py/issues/3710

Reply:
Thanks @archmoj I've added an  example for "monthly period labels with weekly minor ticks".